src/FakerDotNet/Data/CreditCardData.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FakerDotNet.Data
+{
+    internal static class CreditCardData
+    {
+
```

Comment:
Could you also remove these empty lines?

Reply:
Removed Empty lines using extension on visual studio 🥇